### PR TITLE
Add python sdk schema edit references

### PR DESCRIPTION
**Pull request by kostasb**
Add documentation reference and example about updating schema from the Python SDK.

Clarifies [Discord](https://discord.com/channels/996791218879086662/1166252170283339796/1166252170283339796) question.

### CODE REVIEW

File: 040-SDK/020-Python/020-examples.mdx [hunk 1]
```diff
@@ -31,6 +31,7 @@ Below are the parameters passed to the methods in the following examples:
 - `xata.data().ask()`: table name (str), question (str)
 - `xata.data().ask_follow_up()`: table name (str), session id (str), question (str)
 - `to_rfc3339()`: dt (datetime), tz (timezone, default: utc)
+- `xata.table().set_schema()`: table name (str), table schema (dictionary)
```
Comment (philkra) [inline 040-SDK/020-Python/020-examples.mdx:34]:
```suggestion
- `xata.table().create()`: table name (str)
- `xata.table().set_schema()`: table name (str), table schema (dictionary)
```

File: 040-SDK/020-Python/020-examples.mdx [hunk 2]
```diff
@@ -479,3 +480,36 @@ tz_europe_vienna = timezone("Europe/Vienna")
...
+      {
+          "name": "description",
+          "type": "text",
+          "notNull": True,
+          "defaultValue": "Default text"
+      }
+  ]
+}
+
+xata.table().create("mytable")
```
Comment (philkra) [inline 040-SDK/020-Python/020-examples.mdx:509]:
```suggestion
assert xata.table().create("mytable").is_success()
```

File: 040-SDK/020-Python/020-examples.mdx [hunk 3]
```diff
@@ -479,3 +480,36 @@ tz_europe_vienna = timezone("Europe/Vienna")
...
+
+set_table_schema_resp = xata.table().set_schema("mytable", table_schema)
```
Comment (philkra) [inline 040-SDK/020-Python/020-examples.mdx:511]:
```suggestion
resp = xata.table().set_schema("mytable", table_schema)
```

File: 040-SDK/020-Python/020-examples.mdx [hunk 4]
```diff
@@ -479,3 +480,36 @@ tz_europe_vienna = timezone("Europe/Vienna")
...
+
+if set_table_schema_resp.is_success():
```
Comment (philkra) [inline 040-SDK/020-Python/020-examples.mdx:513]:
```suggestion
# if the operation failed, print the server response
assert not resp.is_success(), resp
```

File: 040-SDK/020-Python/020-examples.mdx [hunk 5]
```diff
@@ -479,3 +480,36 @@ tz_europe_vienna = timezone("Europe/Vienna")
...
+  print("Table schema has been updated")
```
Comment (philkra) [inline 040-SDK/020-Python/020-examples.mdx:514]:
```suggestion
```

**Review by philkra — APPROVED** [review]
Overall correct, but left a couple of stylistic changes, can go in as is or with the changes. LGTM and thanks for putting this together

**Comment by kostasb** [comment]
Thanks for the suggestions @philkra all committed. I did a minor adjustment in the last assert to remove the `not` negation.

**Comment by philkra** [comment]
Oh, yikes! Yes, you are correct, the `not` was incorrect. Thank you

**Pull request merged**